Gems/Atom/Feature/Common/Assets/Shaders/Reflections/ReflectionScreenSpaceComposite.azsl

```diff
@@ -81,63 +85,41 @@ float3 SampleReflection(float2 reflectionUV, float mip, float depth, float3 norm
         }
     }
     
-    float3 reflection = PassSrg::m_reflection.SampleLevel(PassSrg::LinearSampler, reflectionUV + closestOffsetUV, mip).rgb;
+    float3 reflection = PassSrg::m_reflection.SampleLevel(PassSrg::LinearSampler, reflectionUV + closestOffsetUV, 0).rgb;
     return reflection;
 }
 
 // Pixel Shader
 PSOutput MainPS(VSOutput IN, in uint sampleIndex : SV_SampleIndex)
 {
-    float2 screenCoords = IN.m_position.xy;
-
-    uint2 dimensions;
-    uint samples;
-    PassSrg::m_depth.GetDimensions(dimensions.x, dimensions.y, samples);
-
-    // compute trace image coordinates for the half-res image
-    float2 traceCoords = screenCoords * 0.5f;
+    float2 fullScreenCoords = IN.m_position.xy;
+    float2 coords = fullScreenCoords * PassSrg::m_outputScale;
+    uint2 invDimensions = 1.0f / (uint2(PassSrg::m_outputWidth, PassSrg::m_outputHeight) * PassSrg::m_outputScale);
```

Comment:
do we need to protect for 1/0 here?

Reply:
m_outputScale should never be zero here, it's set to either 1.0 or 0.5 depending on editor settings.